e2e/netshaper/suite_setup_test.go

```diff
@@ -13,6 +14,14 @@ import (
 type Suite struct {
 	suite.Suite
 	Knuu *knuu.Knuu
+
+	cleanupMu     sync.Mutex
+	totalTests    atomic.Int32
+	finishedTests int32
+}
+
+func TestRunSuite(t *testing.T) {
+	suite.Run(t, new(Suite))
 }
```

Comment:
Follow Up: Can this be refactored to just be in the e2e package? We should be able to use the same struct across the tests as opposed to redefining it.

Reply:
Good idea, yeah let's track it in a followup issue.